packages/plugins/RSS3/src/SiteAdaptor/FeedFilters.tsx

```diff
@@ -0,0 +1,94 @@
+import { Icons } from '@masknet/icons'
+import { makeStyles } from '@masknet/theme'
+import { Box, Typography, type BoxProps } from '@mui/material'
+import { forwardRef, memo } from 'react'
+import { NetworkOptions, Networks } from '../constants.js'
+import { useRSS3Trans } from '../locales/i18n_generated.js'
+import { useFilters } from './filters.js'
+
+const useStyles = makeStyles()((theme) => ({
+    header: {
+        display: 'flex',
+        justifyContent: 'space-between',
+        alignItems: 'flex-start',
+    },
+    button: {
+        flexShrink: 0,
+        display: 'flex',
+        alignItems: 'center',
+        padding: theme.spacing(0.5, 1),
+        height: 24,
+        boxSizing: 'border-box',
+        borderRadius: 8,
+        gap: theme.spacing(0.5),
+        border: `1px solid ${theme.palette.maskColor.secondaryLine}`,
+        cursor: 'pointer',
+        fontSize: 13,
+        fontFamily: 'Helvetica',
+        lineHeight: '18px',
+    },
+    container: {
+        display: 'flex',
+        flexDirection: 'column',
+        gap: 10,
+        padding: theme.spacing(1.5),
+        boxShadow: '0px 0px 20px rgba(0, 0, 0, 0.05)',
+        backdropFilter: 'blur(8px)',
+    },
+    networks: {
+        display: 'flex',
+        gap: theme.spacing(1, 1.5),
+        flexWrap: 'wrap',
+    },
+}))
+
+export const FeedFilters = memo(
+    forwardRef<HTMLDivElement, BoxProps>(function FeedFilters(props: BoxProps, ref) {
```

Comment:
forwardRef has been deprecated. see https://react.dev/blog/2024/04/25/react-19#ref-as-a-prop

Reply:
But mui is not compatible with that yet.

Reply:
<img width="1015" alt="image" src="https://github.com/user-attachments/assets/93cfcfc4-bcd9-4e6f-9725-6ea123c619fb">

Reply:
it should be like this:

```js
// old
export default forwardRef<HTMLDivElement, BoxProps>(function (props: BoxProps, ref) {})

// new
export default function({ ...props, ref }: BoxProps & { ref: ... }) {}
```

if this does not work, that will be strange